### PR TITLE
Use a FQDN in the reserved_domain example

### PR DESCRIPTION
**Pull request by jrobsonchase**

### CODE REVIEW

File: examples/reserved_domain.rs [hunk 1]
```diff
@@ -17,7 +17,7 @@ async fn main() {
     let rand: u64 = rand::thread_rng().gen();
     let resp = rd
         .create(&types::ReservedDomainCreate {
-            name: format!("rustexample{}", rand),
+            name: format!("rustexample{}.ngrok.io", rand),
```
Comment (marcelchampagne) [inline examples/reserved_domain.rs:20]: name: -> domain:

**Review by marcelchampagne — APPROVED** [review]
:bow: thanks!

**Review by euank — APPROVED** [review]

**Comment by marcelchampagne** [comment]
I imagine this is broken because we haven't released the rust API client libraries since the API changes to add 'domain'? 

FYI that this isn't high priority, I've decided to punt on actually marking the `name` field as deprecated to avoid any breaking changes to the API client libraries for now. I've fine if you want to close out the PR and ignore this for now. I created a tracking issue to do the whole thing eventually: https://github.com/ngrok-private/ngrok/issues/15591.

**Comment by andrewryno** [comment]
Fixed in https://github.com/ngrok/ngrok-api-rs/pull/11.

**Pull request closed without merging**